src/common/transformations/include/transformations_visibility.hpp

```diff
@@ -12,7 +12,6 @@
  */
 
 /**
- * @ingroup ie_cpp_api
  * @defgroup ie_transformation_api Inference Engine Transformation API
  * @brief Defines Inference Engine Transformations API which is used to transform ngraph::Function
```

Comment:
Should we replace "Inference Engine" with "OpenVINO Runtime" everywhere?

Reply:
yes, we should. We will go step by step :)